### DISCUSSION TITLE
Is there a way to separate individual instruments?

### DISCUSSION THREAD

**Discussion by Vexxi666** [Anjok07/ultimatevocalremovergui#1136] [Q&A]
Hi!

Is there a way to separate individual instruments?
I'm looking to separate individual instruments in very thick metal mixes with a lot of layers going on(most importantly guitars as I'm trying to train my ear), but figuring out what layers were used also helps me understand production better.

Thanks!

**Comment by ghost** [comment]
AI-based separation are not performing that well on thick metal mixes in general, and there's no public models for guitars atm.
But as it's for learning, you should look at VST plugins like Mashtatic, which allow to isolate sounds/parts based on their stereo placement, EQ and dynamics. With metal guitar you should get most of them on the extreme sides.

**Comment by luca-brwn** [comment]
there is a model under demucs called 6s, which separates piano, guitar, and other. but right now there isn't a specific model for individual stems. there is a cool and old program called [isse](https://isse.sourceforge.net/), it's really good at separating drums and certain synths and textures, but it has not been updated in a while so compatibility might be an issue.
